pkg/network/proxy.go

```diff
@@ -16,14 +16,17 @@ import (
 
 // ShouldDeployKubeProxy determines if the desired network type should
 // install kube-proxy.
-// openshift-sdn and OVN-Kubernetes deploy their own kube-proxy. All other
+// openshift-sdn and OVN-Kubernetes deploy their own kube-proxy.
+// Kuryr-Kubernetes handles services on its own. All other
```

Comment:
Actually, the existing text is wrong about ovn-kubernetes. It should say "OpenShift SDN deploys its own kube-proxy. OVN Kubernetes and Kuryr-Kubernetes handle services on their own".